src/systems/systems/src/hydraulic/brake_circuit.rs

```diff
@@ -402,10 +403,7 @@ impl Default for AutobrakePanel {
 /// Deceleration governor is the PI controller computing the expected brake force to reach the target
 /// it's been given by update caller
 pub struct AutobrakeDecelerationGovernor {
-    target: Acceleration,
-    i_gain: f64,
-    p_gain: f64,
-    last_error: f64,
+    pid_controller: Pid,
```

Comment:
**thought**: It would be really awesome if we could write `Pid<Acceleration>`, such that we don't have the `.get::<meter_per_second_squared>()` calls everywhere. Sadly, uom's macro generic types are not so trivial and I've been unable to figure out which generic type constraints would need to be applied to the `Pid` type for it to work. Maybe @devsnek knows?

Reply:
@devsnek Something _like_ that. But that isn't really the type I need. I think it's something along the lines of `Quantity<Dimension<???>, ???, ???>` but am not sure how to go from there.

> the trait bound `Quantity<(dyn uom::si::Dimension<L = PInt<UInt<UTerm, B1>>, J = Z0, M = Z0, T = NInt<UInt<UInt<UTerm, B1>, B0>>, N = Z0, I = Z0, Kind = (dyn Kind + 'static), Th = Z0> + 'static), (dyn Units<f64, electric_current = ampere, time = uom::si::time::second, luminous_intensity = candela, mass = kilogram, thermodynamic_temperature = uom::si::thermodynamic_temperature::kelvin, amount_of_substance = mole, length = meter> + 'static), f64>: uom::si::Unit` is not satisfied
the trait `uom::si::Unit` is not implemented for `Quantity<(dyn uom::si::Dimension<L = PInt<UInt<UTerm, B1>>, J = Z0, M = Z0, T = NInt<UInt<UInt<UTerm, B1>, B0>>, N = Z0, I = Z0, Kind = (dyn Kind + 'static), Th = Z0> + 'static), (dyn Units<f64, electric_current = ampere, time = uom::si::time::second, luminous_intensity = candela, mass = kilogram, thermodynamic_temperature = uom::si::thermodynamic_temperature::kelvin, amount_of_substance = mole, length = meter> + 'static), f64>`rustcE0277